esmvalcore/config/extra_facets/native6-mappings.yml

```diff
@@ -0,0 +1,192 @@
+# Extra facets for native6 data
+
+# Notes:
+# - All facets can also be specified in the recipes. The values given here are
+#   only defaults.
+
+# A complete list of supported keys is given in the documentation (see
+# ESMValCore/doc/quickstart/find_data.rst).
+---
+
+ERA5:
+
+  # Settings for all variables of all MIPs
+  '*':
+    '*':
+      family: E5
```

Comment:
Is there a place where the meaning of these facets is explained? If yes, it would be nice to add a link here to make it easier to understand where they come from and how to update them.

Reply:
Added in https://github.com/ESMValGroup/ESMValCore/pull/2178/commits/4c3f1beb0e55fa70d0e7c9bdfcc1096231ecce12